ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/UcteExporter.java

```diff
@@ -86,6 +96,13 @@ public void export(Network network, Properties parameters, DataSource dataSource
 
         UcteNetwork ucteNetwork = createUcteNetwork(network, namingStrategy);
 
+        String postProcessorScript = ConversionParameters.readStringParameter(getFormat(), parameters, POST_PROCESSOR_PARAMETER, defaultValueConfig);
+        if (postProcessorScript != null) {
+            Path script = config.getConfigDir().resolve(postProcessorScript);
+            ExportPostProcessor processor = new GroovyScriptExportPostProcessor(script);
+
+            processor.process(network, null, ucteNetwork, null);
+        }
```

Comment:
The list of the PostProcessor to run should be passed as parameters to this method or discovered automatically. Passing these information to the method is a breaking change, so be carful: you have to manage the compatibility with a default behavior to the new method. But I think it's the proper way to do it... otherwise we could not pass an explicit list of post-processors
@geofjamg What do you think?

Reply:
Yes, we can pass the list of post processor to the export method and with default methods it should be easy to get backward compatibility.